src/website/app/demos/Tabs/examples/Tabs/alignment.js

```diff
@@ -0,0 +1,46 @@
+/* @flow */
+import { Fragment } from 'react';
+import { CardDivider } from '../../../../../../library/Card';
+import Tabs, { Tab } from '../../../../../../library/Tabs';
+import Text from '../../../../../../library/Text';
+import DemoLayout from '../../../shared/DemoLayout';
+import content from '../../shared/content';
+
+export default {
+  id: 'alignment',
+  title: 'Tabs Alignment',
+  description: `Control the alignment of the list of [Tabs](/components/tab)
+with the \`align\` prop. Note that a value of \`'justify'\` will force all Tabs 
+to be equal width and [\`maxTabWidth\`](#max-tab-width) will not apply.`,
+  scope: { CardDivider, content, DemoLayout, Fragment, Tabs, Tab, Text },
+  source: ` () => {
+    const alignments = ['start', 'center', 'end', 'justify'];
+
+    const tabs = ({ position } = {}) => alignments.map((alignment, index) => {
```

Comment:
FYI: When the position is 'start' or 'end' and the window is wide, one cannot differentiate the alignments as there is not enough height for them to appear different.  Not sure if you want to worry about this or not.